jenkins/ci.suse.de/pipelines/openstack-ardana.Jenkinsfile

```diff
@@ -16,328 +16,230 @@ pipeline {
   }
 
   stages {
-    stage('setup workspace and environment') {
+    stage('Setup workspace') {
       steps {
         cleanWs()
-
         script {
           env.cloud_type = "virtual"
-          if ( ardana_env == '') {
+          if (ardana_env == '') {
             error("Empty 'ardana_env' parameter value.")
           }
-          currentBuild.displayName = "#${BUILD_NUMBER} ${ardana_env}"
           if ( ardana_env.startsWith("qe") || ardana_env.startsWith("qa") ) {
```

Comment:
One last request, if you have the time for it: instead of using this technically incorrect check to determine if the the setup is hardware or virtual, could you check the `ardana_env` value against the ansible inventory using a shell command and save the result in the `cloud_type` environment variable ?

Reply:
As we discussed, using pipe inside groovy `sh` is tricky and it will require a bit more time for testing. So, for now we are leaving it as it always has been.